flask_inertia/inertia.py

```diff
@@ -43,16 +45,19 @@
 from flask_inertia.version import get_asset_version
 from flask_inertia.views import render_inertia
 
+# TODO: maybe `Scaffold` instead?
+FlaskOrBlueprint = Union[Flask, Blueprint]
```

Comment:
Since Python 3.10 `typing`  supports and recommends the shorthand `Flask | Blueprint `. I think you can use this since  it has been released for 3 years now. Also I'm not really sure you need to declare it as a new type. I find the following notation totally fine

```python
def __init__(self, app: Optional[Flask | Blueprint] = None):
```

Reply:
Wasn't sure what your minimum supported python version is, which is why I went with `Union`. Technically python 3.9 is not end-of-life yet but up to you if you're supporting it.

Reply:
[988f051](https://github.com/j0ack/flask-inertia/pull/25/commits/988f0513eb16b41ab9d27ea8249bdc38b76ad427)